src/mixer.c

```diff
@@ -941,7 +941,7 @@ int libxmp_mixer_on(struct context_data *ctx, int rate, int format, int c4rate)
 	s->format = format;
 	s->amplify = DEFAULT_AMPLIFY;
 	s->mix = DEFAULT_MIX;
-	/* s->pbase = C4_PERIOD * c4rate / s->freq; */(void) c4rate;
+	/* s->pbase = C4_PERIOD * c4rate / s->freq; */
```

Comment:
Likely to have been to suppress `-Wunused-parameter`? Which isn't a *totally* useless warning, but libxmp rightly turns it off now.

Reply:
indeed